src/main/java/games/strategy/engine/data/CompositeRouteFinder.java

```diff
@@ -15,7 +16,7 @@
   private static final Logger s_logger = Logger.getLogger(CompositeRouteFinder.class.getName());
 
   private final GameMap m_map;
-  private final HashMap<Match<Territory>, Integer> m_matches;
+  private final Map<Match<Territory>, Integer> m_matches;
```

Comment:
Looks like you are correct. I would definitely probably change how this is structured. It essentially could just be a static method call and only used in 1 place.